bigtable/examples/run_examples_utils.sh

```diff
@@ -155,8 +196,8 @@ function run_all_data_examples {
   local -r TABLE="sample-table"
 
   # Create an instance to run these examples.
-  $admin ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
-  trap '$admin ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}" "${zone_id}"' EXIT
+  $setenv ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
+  trap 'exit_handler ${project_id} ${INSTANCE}' EXIT
```

Comment:
Is this using single quotes to save the variable references textually and let them resolve later? Single quotes don't expand env vars, correct?

Reply:
Correct.  Well, I could evaluate immediately, but that would be a mess of escaped quotes:

```sh
trap "exit_handler \"${project_id}\" \"${INSTANCE}\"" EXIT
```

Or we could create a function inline:

```sh
trapper() { exit_handler "${project_id}" "${INSTANCE}"; }
trap trapper EXIT
```

all of these work (if I recall my tests correctly), but the code I wrote seemed the simplest to me (I could be wrong though).